EndpointHandler.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+require "settings.php";
+
+// Decodes the data array and returns the value of "claim"
+function getClaim($claim, $data) {
+	
+	$data_array = json_decode($data, true);
+	return $data_array[$claim];
+}
```

Comment:
I think it made more sense to decode the data just once. Because of that, I removed the function entirely, and didn't create a separate class.

---
In reply to: [71198565](https://github.com/AzureAD/active-directory-b2c-wordpress-plugin-openidconnect/pull/3#discussion_r71198565) [](ancestors = 71198565)